bin/remotelookup.js

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
+
+const http = require("http");
+const axios = require("axios");
+
```

Comment:
This file is the same as testlookup.js, neither does remote (trust) lookup. What is this for? And where's remote lookup?

Reply:
they are calling different endpoints

remote lookup is calling trust/lookup that is doing the remote lookup